onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3072,7 +3072,12 @@ def aten_logspace(start: float, end: float, steps: int, base: float = 10.0) -> T
 def aten_logsumexp(self: TReal, dim: INT64, keepdim: int = False) -> TReal:
     """logsumexp(Tensor self, int[1] dim, bool keepdim=False) -> Tensor"""
 
-    return op.ReduceLogSumExp(self, dim, keepdims=keepdim)
+    if op.Size(op.Shape(self)) == 0:
+        # A scalar
+        result = op.Identity(self)
```

Comment:
Why do we need to add op.Identity here? What's the different between it and returning self directly?

Reply:
removed

Reply:
they should be the same